src/iptvsimple/utilities/FileUtils.h

```diff
@@ -12,6 +12,15 @@
 
 #include <string>
 
+// Prevent conflicts with Windows macros where have this names.
+#ifdef CreateDirectory
+#undef CreateDirectory
+#endif // CreateDirectory
+#ifdef DeleteFile
+#undef DeleteFile
+#endif // DeleteFile
+#endif // _WIN32
```

Comment:
http://ix.io/2mEz

Reply:
Ups

Reply:
Have added the missing `#ifdef`, this was moved as them to new headers and not seen by global jenkins test that there needed too, thats why now on addon.